cloudbuild.yaml

```diff
@@ -16,6 +16,14 @@ steps:
   secretEnv: ['FIREBASE_TOKEN']
   waitFor: ['-']
 
+- name: 'gcr.io/solo-public/go-mod-make:0.1.1'
```

Comment:
I thought we are planning to not deploy the new docs till we finish the refactor?